msg/DifferentialPressure.msg

```diff
@@ -1,10 +1,11 @@
-uint64 timestamp                     # time since system start (microseconds)
-uint64 timestamp_sample
+# Differential-pressure sensor
+#
+# ?
 
-uint32 device_id                     # unique device ID for the sensor that does not change between power cycles
+uint64 timestamp         # [us] Time since system start
+uint64 timestamp_sample  # [us] Timestamp of the raw data
```

Comment:
How does that differ from the timestamp?

Reply:
The `timestamp` is publication metadata (ideally), the `timestamp_sample` should be as close to the physical sample as possible. For example with many of the IMUs there's a data ready interrupt (that goes in the `timestamp_sample`) then a relatively large transfer of the FIFO, then publication of sensor_accel/sensor_gyro where `timestamp` is set on publication.